README.md

```diff
@@ -15,6 +15,10 @@ For Mac users, I highly recommend iTerm 2 + Solarized Dark
 
 **NOTE:** In all likelihood, you will need to install a [Powerline-patched font](https://github.com/Lokaltog/powerline-fonts) for this theme to render correctly.
 
+For iTerm 2 users, you have to set the font : iTerm 2 -> Preferences -> Profiles -> Text and use one of the Meslo fonts :
+
+<img src="https://gist.github.com/mgohin/0144c7930a3ff81f86f7b3bfb06558a1/raw/d8548fb86788e26900df3be43be6e064af616a87/iterm_set_font.png" height="150">
```

Comment:
I would rather use a bigger screenshot (showing we can update this preference in iterm2 in "Text" tab)

Something like : https://raw.githubusercontent.com/fcamblor/oh-my-zsh-agnoster-fcamblor/master/iterm_set_font.png